src/cdi_analytic/test/tstAnalytic_CP_Eloss.cc

```diff
@@ -29,9 +29,9 @@ using rtt_cdi_analytic::Analytic_Eloss_Model;
 void KP_alpha_test(rtt_dsxx::UnitTest &ut) {
 
   // Deuterium:
-  int32_t target_in = 1002;
+  rtt_cdi::CParticle target_in(1002, 3.34358e-24);
```

Comment:
Use an appropriately named variables to capture this int and double value to make it easier for this test to be used as a tutorial (self-documenting).

Reply:
Fixed (here and in `cdi_CPEloss/test/tstTabular_CP_Eloss.cc`)